packages/sn-search-react/src/Components/Fields/ReferenceField.tsx

```diff
@@ -177,7 +177,7 @@ export function ReferenceField<T extends GenericContent>(props: ReferenceFieldPr
           displayName={displayName}
           description={description}
           helperText={helperText}
-          inputProps={inputComponentProps}
+          inputProps={inputComponentProps as any}
```

Comment:
i think `as any` is not necessary

Reply:
:'(
![image](https://user-images.githubusercontent.com/22347393/109815681-338bb380-7c30-11eb-8afa-9003e875211f.png)